Makefile

```diff
@@ -403,6 +403,11 @@ cluster/prepare/sts:
 	@-oc process -n $(NAMESPACE_PREFIX)cloud-resources-operator NAME=sts-credentials NAMESPACE=$(NAMESPACE_PREFIX)cloud-resources-operator ROLE_ARN=$(CRO_ROLE_ARN) -f config/secrets/sts-secret.yaml | oc apply -f -
 	@-oc process -n $(NAMESPACE_PREFIX)3scale NAME=sts-s3-credentials NAMESPACE=$(NAMESPACE_PREFIX)3scale ROLE_ARN=$(THREESCALE_ROLE_ARN) -f config/secrets/sts-secret.yaml | oc apply -f -
 
+
+.PHONY: cluster/prepare/addon-instance
+cluster/prepare/addon-instance:
+	@-oc apply -f config/samples/addoninstance_v1alpha1.yaml
```

Comment:
Do we need to update the `make cluster/cleanup` target to remove this as well?

Reply:
No, the addon instance will be deleted when the namespace is deleted in `make cluster/cleanup`